keyboards/handwired/erikpeyronson/erkbd/keyboard.json

```diff
@@ -0,0 +1,116 @@
+{
+    "manufacturer": "Erik Peyronson",
+    "keyboard_name": "erkbd",
+    "maintainer": "erikpeyronson",
+    "url": "https://github.com/erikpeyronson/erkbd",
+    "bootloader_instructions": "Double tap the reset button on the controller board if accessable. If not hold down bootmagic key while powering on. Left side: row 0 column 0. (The tab key on regular keyboard). Right side: row 5 column 4 (the Backspace key on regular keyboard)",
+    "usb": {
+        "vid": "0xFEED",
+        "pid": "0x1234",
+        "device_version": "1.0.0"
+    },
+    "features": {
+        "encoder": true,
+        "bootmagic": true,
+        "oled_enable": true,
+        "oled": true
```

Comment:
If OLEDs are enabled at the keyboard level, there should exist some keyboard-level configuration for them in the `erkbd.c` file.